src/clientagent/AstronClient.cpp

```diff
@@ -375,10 +407,12 @@ class AstronClient : public Client, public NetworkClient
             NetworkClient::send_disconnect();
         }
         break;
-        case CLIENT_OBJECT_SET_FIELD: {
```

Comment:
I haven't consulted the style guide, although honestly I think braces looks better (and is probably more correct). Furthermore, be consistent; if you undo this change, also add braces to the heartbeat clause :smile: